.ci-cd/Dockerfile.cpu

```diff
@@ -82,14 +82,16 @@ RUN pip3 install git+https://github.com/HorizonRobotics/gin-config.git
 COPY requirements.txt  /tmp/requirements.txt
 RUN pip3 install -r /tmp/requirements.txt
 
-# import public ROM for super mario (ignore shasum)
-RUN cd /tmp \
-    && wget \
-      'https://static.roms.download/roms/nintendo/Super Mario Bros (E).zip' \
-    && unzip 'Super Mario Bros (E).zip' \
-    && mv 'Super Mario Bros (E).nes' \
-      /usr/local/lib/python3.6/dist-packages/retro/data/stable/SuperMarioBros-Nes/rom.nes \
-    && rm 'Super Mario Bros (E).zip'
+# The source is inaccessible anymore. We leave it here as a reference and
+# install it manually when building the image if needed.
+# # import public ROM for super mario (ignore shasum)
+# RUN cd /tmp \
```

Comment:
FYI, our internal Gitlab alf_k8s has a checked-in image for this. 

http://gitlab.hobot.cc/hobotrl/docker_build/alf_k8s/raw/pytorch/alf/package/retro/SuperMarioBros.nes?inline=false

Reply:
> We have too many docker images for CI, local usage, and cluster usage. I think probably a documentation page will help.

Will do. But probably not put it here, maybe as an internal doc?
 
> Also, the update for cluster dockers is missing.

I'm aware of this and working on it. Since it is in different repos, it is not reflected here.

Reply:
> 
> Will do. But probably not put it here, maybe as an internal doc?

A Feishu doc would be good.

Reply:
Here is a doc:
https://oshcjymfmk.feishu.cn/docs/doccnr4Qg0kb55mIU2ueG7wYs9d